### PR TITLE
5 view category page

### PR DESCRIPTION
**Pull request by fuzzyJess**
Categories page now displaying buttons to click which take you through to reviews matching that category.

### CODE REVIEW

File: src/components/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  useEffect(() => {
+    setIsLoading(true)
+    api.fetchCategories().then((response) => {
+        setCategories(response);
+        setIsLoading(false);
+          // poss if (>200){deal with diff errors}
+        setErr(null)
+    }).catch((err) => {
+        setErr(err)
```
Comment (alexdevdavis) [inline src/components/Categories.jsx:20]: You'll need to set isLoading again in catch block in order to hit line 27

File: src/components/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <>
+    <h2>Game categories</h2>
+        <main>
+          
+          <table id="cat_table">
+              {categories.map((category) => {
+                  return (
+                  <tr>
```
Comment (alexdevdavis) [inline src/components/Categories.jsx]: this tr element is what needs a unique key (React demands the parent element of each list item holds the key)

File: src/components/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    const {category} = useParams();
+   
+    useEffect(() => {
+        setIsLoading(true)
+        api.fetchReviews(category).then((reviews) => {
+        setGameCategory(reviews);
+        setIsLoading(false);
+        setErr(null)
+    }).catch((err) => {
+        setErr(err)
```
Comment (alexdevdavis) [inline src/components/Category.jsx:20]: see previous isLoading comment

**Review by alexdevdavis — APPROVED** [review]
Nicely done, PR approved. 
A couple of comments to consider before getting too deep into the next ticket.

File: src/components/Categories.jsx (hunk 2 above)
Comment (fuzzyJess) [reply src/components/Categories.jsx]: Changed :-)

**Pull request merged**